routes/root.js

```diff
@@ -26,14 +26,15 @@ module.exports = [
 
       try {
         // We might have had a state or nonce manually passed through in our url
-        const { state, nonce, scope, serviceId, stubScp, prompt } = query
+        const { state, nonce, scope, serviceId, stubScp, prompt, _ga } = query
 
         let outboundUrl = await server.methods.idm.generateOutboundRedirectUrl(request, request.query, {
```

Comment:
Actually, the _ga param goes into the second argument.
So it will just go in via `request.query`

But did you not want to add a custom value to it so we know what consuming service they're using?